web/html/src/scripts/docker-javascript-lint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+### TODO remove this section before merging PR. it's just for testing in jenkins
+zypper ar -f http://download.suse.de/ibs/home:/mateialbu:/branches:/Devel:/Galaxy:/Manager:/Head/SLE_12_SP3 "nodejs-sdk"
+
+zypper ar -f http://download.suse.de/ibs/Devel:/Galaxy:/nodejs/SLE_12_SP3/ "nodejs"
+
+zypper --non-interactive in nodejs8 npm8
+
+zypper --non-interactive in --repo "nodejs-sdk" susemanager-nodejs-sdk-devel
+
+# install flow-bin globally
+npm_config_prefix=/usr npm install -g flow-bin@0.73
```

Comment:
Did you try the last version 0.81? if it doesn't bring new errors I think it would make sense to start with the most recent one